content/concepts/transports/quic.md

```diff
@@ -95,8 +95,26 @@ IDs are authenticated in the
 
 {{< alert icon="💡" context="note" text="To be clear, there is no additional security handshake and stream muxer needed as QUIC provides all of this by default. This also means that establishing a libp2p connection between two nodes using QUIC only takes a single RTT." />}}
 
-Following the multiaddress format described earlier, a standard QUIC connection will
-look like: `/ip4/127.0.0.1/udp/65432/quic/`.
+Following the multiaddress format, a standard QUIC connection will
+look like: `/ip4/127.0.0.1/udp/65432/quic-v1/`.
+
+### QUIC code points
+
+The initial implementation of QUIC of go-libp2p was based on
+[draft-ietf-quic-transport-29](https://datatracker.ietf.org/doc/html/draft-ietf-quic-transport-29)
+(or simply, **draft-29**), because [RFC 9000](https://datatracker.ietf.org/doc/html/rfc9000)
+was yet to be finalized. Some legacy nodes on the IPFS network still use draft-29.
+By using different code points (`quic-v1` for RFC 9000 and `quic` for draft-29),
+we can distinguish between the two versions.
```

Comment:
```suggestion
(or simply, **draft-29**). At the time, draft-29 was implemented because [RFC 9000](https://datatracker.ietf.org/doc/html/rfc9000)
was yet to be finalized.
Eventually go-libp2p added support for RFC 9000 in addition to draft-29, supporting two QUIC versions.
However, the multiaddresses for these versions used the same format and thus were indistinguishable.

By using different code points, `quic-v1` for RFC 9000 and `quic` for draft-29,
libp2p can distinguish between the two versions.
```

Reply:
`Some legacy nodes on the IPFS network still use draft-29.`
removed this sentence because libp2p is product agnostic and should not mention IPFS.